app/src/main/java/com/codesoom/models/SendResponseData.java

```diff
@@ -0,0 +1,56 @@
+package com.codesoom.models;
+
+import com.codesoom.HttpEnum.HttpStatusCode;
+
+/**
+ *  SendResponseData 클래스는 HttpExchange sendResponseHeaders 메소드에 들어가는 매개변수를 담고 있는 클래스입니다.
+ * @author HyoungUkJJAng(김형욱)
+ */
+public class SendResponseData {
+
+    /**
+     * HTTP 상태코드를 저장하는 변수입니다.
```

Comment:
private 멤버에는 굳이 javadoc을 작성하지 않아도 괜찮습니다.

굳이 작성한다면 다음과 같이 작성하는 것이 더 좋다고 생각해요. 변수를 설명할 때 "~변수입니다."하는 것은 안타깝죠.

```suggestion
     * HTTP 상태코드.
```